client/src/hooks/useChangeView.js

```diff
@@ -0,0 +1,15 @@
+import {useEffect} from 'react';
+import {useLocalStorage} from './useLocalStorage';
+
+export const useChangeView = (key, initVal) => {
```

Comment:
It would also be nice to have a comment here to know what change view is supposed to do.